tests/Fixtures/RestaurantFixture.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * Main File with fixtures for test entities.
+*/
+
+namespace Tests\Fixtures;
+
+
+use AppBundle\Entity\Restaurant;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+
+class RestaurantFixture extends AbstractFixture implements DependentFixtureInterface {
+
+    public function load(ObjectManager $manager)
+    {
+        $restaurant = new Restaurant();
+        $restaurant->setName('Test Restaurant');
+        $restaurant->setAddress($this->getReference('restaurant-address'));
+
+        $manager->persist($restaurant);
+        $manager->flush();
+
+    }
+
+    public function getDependencies()
+    {
+        return array(AddressFixture::class);
+    }
+}
+
+
+?>
```

Comment:
Useless.

Reply:
Yeah, was planning it to use it for some functional tests. I can remove for now on.

Reply:
I was talking about the closing PHP tag 🙂